anomalib/models/ganomaly/torch_model.py

```diff
@@ -40,13 +42,14 @@ def __init__(
     ):
         super().__init__()
 
-        assert input_size % 16 == 0, "Input size should be a multiple of 16"
+        assert input_size[0] % 16 == 0 and input_size[1] % 16 == 0, "Input size should be a multiple of 16"
```

Comment:
I guess this is no longer needed now that we use padding to make sure that the network can process any image size.